### DISCUSSION TITLE
Is this project abandoned?

### DISCUSSION THREAD

**Discussion by sciotta** [roginfarrer/system-props#71] [Q&A]
Hello! Do you want to keep acting and coding this project? I appreciate the consistency and simplicity of features. It's perfect for anyone who is building a design system.

**Comment by roginfarrer** [comment]
Hey there, I'm not actively developing this project. I'm no longer using it any of my own projects since I've mostly moved away from styled-components and emotion. (You might be interested in my more recent project, [rainbow-sprinkles](https://github.com/wayfair/rainbow-sprinkles), which adds this functionality to Vanilla Extract.) That said, I'm happy to review contributions if others want to work on it.
